src/ert/gui/simulation/ensemble_experiment_panel.py

```diff
@@ -31,7 +30,13 @@ class Arguments:
 
 
 class EnsembleExperimentPanel(ExperimentConfigPanel):
-    def __init__(self, ensemble_size: int, run_path: str, notifier: ErtNotifier):
+    def __init__(
+        self,
+        analysis_config: AnalysisConfig,
```

Comment:
Do we need the entire analysisconfig, or only designmatrix?

Reply:
Good question and I have thought about it and decided on using the entire config:
 - other panels get the entire analysisconfig too and thus it makes the api more uniform
 - when doing the validation ( not there yet), we might to access more of the configs